packages/forms/stories/json/columns.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
please see my comments here: https://jira.talendforge.org/browse/TDKN-138?focusedCommentId=465162&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-465162
Today this would require a complete rewrite of the serializer in daikon. Let talk about it.